python_modules/dagster-webserver/dagster_webserver_tests/test_app.py

```diff
@@ -90,7 +90,7 @@ def test_index_view(instance):
         res = client.get("/")
 
         assert res.status_code == 200, res.content
-        assert b"<title>Dagster Cloud</title>" in res.content
+        assert b"<title>Dagster</title>" in res.content
```

Comment:
this was a separate PR right?

Reply:
I realized I forgot to update the test in that PR

Reply:
And didn't notice because I saw a buildkite failure that was unrelated and assumed all of them were unrelated 😓